backend/arm64_ast.mli

```diff
@@ -82,8 +82,7 @@ module Instruction_name : sig
       | GT
       | LE
       | AL
-      (* CR sspies: Remove [LO], since it is an alias of [CC] *)
-      | LO
+      | NV
```

Comment:
We should not  emit `NV`, it's behavior is equivalent to `AL` but it has a different encoding. I prefer to not have it in this list (add a comment maybe to explain why it's not here?). An alternative is to assert that it's not printed.